PORTAL/jobs/__main__.py

```diff
@@ -510,7 +510,7 @@ def cmd_queue_push(jobs: Jobs, reqid: RequestID) -> None:
     try:
         pos = jobs.queue.push(reqid)
     except JobAlreadyQueuedError:
-        for pos, queued in enumerate(jobs.queue, 1):
+        for pos, queued in enumerate(jobs.queue):
```

Comment:
IIRC, I intentionally used 1-based indexing for logged messages.

Reply:
Got it.  Reverted.